src/main/kotlin/net/atos/zac/smartdocuments/rest/RESTSmartDocumentsValidator.kt

```diff
@@ -0,0 +1,21 @@
+package net.atos.zac.smartdocuments.rest
+
+import net.atos.zac.smartdocuments.SmartDocumentsException
+
+/**
+ * Validates that all elements in a templates group set are part of pre-defined templates group superset.
```

Comment:
What is this predefined templates group superset? Predefined by whom? Is this part of the acc criteria of the story?

Reply:
This is sanity check validation that should prevent request with structure that is not subset of SmartDocuments'

We're checking if the REST request uses groups an templates with the same IDs and same hierarchy as in SmartDocuments

Got rid of "pre-defined" and added the above sentence.